### DISCUSSION TITLE
All devices connected to co-ordinator?

### DISCUSSION THREAD

**Discussion by noogie1** [Koenkk/zigbee2mqtt#12107] [Devices]
Hey! I have a little question If I may...

I'm running Conbee2 stick with Z2M via a Raspberry Pi. About 40-50 bulbs and sensors
I've paired all my devices and It works great, especially with the new update my hue buttons seem miles quicker to respond! 

I noticed at the back of my house furthest away from the controller the bulbs will sometimes not respond. When looking at the network map I can see *everything* is connected directly to the co-ordinator, but some devices have a dashed line between them. 

Do i need to do something in particular to get my devices meshing properly, like forget and re-pair the bulbs furthest away so they pick a better route? Or do i need to enable certain bulbs to act as repeaters? 

Thanks so much for any help :)

Simon

**Comment by noogie1** [comment]
I've uploaded an image of my network map if it helps:

https://imgur.com/JQUJ1wQ

**Comment by zimbo86** [comment]
did you find any solution?
